puppet/zulip/files/supervisor/conf.d/zulip.conf

```diff
@@ -7,40 +7,20 @@
 ; Note: shell expansion ("~" or "$HOME") is not supported.  Environment
 ; variables can be expanded using this syntax: "%(ENV_HOME)s".
 
-[fcgi-program:zulip-django]
-command=/home/zulip/deployments/current/manage.py runfcgi daemonize=False maxchildren=20 ; the program (relative uses PATH, can take args)
-;process_name=%(program_name)s ; process_name expr (default %(program_name)s)
-;numprocs=1                    ; number of processes copies to start (def 1)
-;directory=/tmp                ; directory to cwd to before exec (def no cwd)
-;umask=022                     ; umask for process (default None)
+[program:zulip-django]
+command=/home/zulip/deployments/current/zulip-venv/bin/uwsgi --ini /etc/zulip/uwsgi.ini
```

Comment:
>Oh, I think the problem is that you're hardcoding the python 2 virtualenv path here, and the way we switch to Python 3 for the production tests involves using #! lines (there's a script tools/replace-tarball-shebang that does this).

So I got confused. I just realized that in Python 3 we get `zulip-py3-venv` and we use `setup_path_on_import` to activate the correct venv. I have created a Django command to run uwsgi. I'll fix this later tonight before going to sleep. I have pushed the commit with the code [here](https://github.com/zulip/zulip/pull/1945). What confuses me is that why the code worked before.